src/lib.rs

```diff
@@ -368,6 +368,29 @@ pub fn ring_commitment(keys: &[u8]) -> Vec<u8> {
 
 const SIGNATURE_SIZE: usize = 784;
 
+/// Derive Private and Public Key from Seed
+///
+/// returns: Vec<u8> containing the exit status followed by the private key followed by the public key
```

Comment:
```suggestion
/// returns: `Vec<u8>` containing the exit status (x bytes) followed by the private key (y bytes) followed by the public key (z bytes).
```

I think adding bytes info will help with parsing the output.